initializr-service/src/main/resources/application.yml

```diff
@@ -1256,6 +1263,17 @@ initializr:
           links:
            - rel: reference
              href: https://docs.spring.io/spring-shell/docs/2.0.0.M2/reference/htmlsingle/
+        - name: Spring Statemachine
+          id: spring-statemachine
+          groupId: org.springframework.statemachine
+          artifactId: spring-statemachine-starter
+          description: Statemachine integration
```

Comment:
Can we have a better description please? Something succinct that explains what the project does.